java-checks/src/main/java/org/sonar/java/checks/PrivateFieldUsedLocallyCheck.java

```diff
@@ -49,6 +56,7 @@
 public class PrivateFieldUsedLocallyCheck extends IssuableSubscriptionVisitor {
 
   private static final String MESSAGE = "Remove the \"%s\" field and declare it as a local variable in the relevant methods.";
+  private static final String QUICK_FIX_MESSAGE = "Move declaration to the relevant method";
```

Comment:
Not sure it is better but as I user I think I would have preferred a message like:
'Move field to the method where it is used'.

Reply:
How about `Move this field to the only method where it is used` ?

Reply:
Knowing that the message from IntelliJ is `Convert this field to a local variable in method '<method name>'`

Reply:
`Move this field to the only method where it is used` is even better. It gives more clue to the user.